### PR TITLE
OPS-3270: Upgrade GoCD instance to Ubuntu 18.04

### PR DESCRIPTION
**Pull request by isabelgk**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/common_vars/defaults/main.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ common_debian_pkgs:
   - ntp
   - acl
   - iotop
-  - lynx-cur
```
Comment (coryleeio) [inline playbooks/roles/common_vars/defaults/main.yml:116]: Did you test that this works in xenial?

Comment (isabelgk) [reply playbooks/roles/common_vars/defaults/main.yml:116]: Was just going to comment that this still needs to be fixed, but trying to decide the least messy way to do it

Comment (isabelgk) [reply playbooks/roles/common_vars/defaults/main.yml:116]: In a clean Xenial Docker container, I was able to apt-get install lynx.

File: playbooks/roles/newrelic_infrastructure/tasks/main.yml [hunk 2]
```diff
@@ -89,6 +101,36 @@
...
+- name: Create runlevel script if doesn't exist
+  file:
+    path: "/etc/rc.local"
+    state: touch
+    owner: root
+    group: root
+    mode: 0755
+  when: script_exists.stat.exists is defined and not script_exists.stat.exists
+
+- name: Add lines to runlevel script if didn't exist
```
Comment (coryleeio) [inline playbooks/roles/newrelic_infrastructure/tasks/main.yml]: We talked about this and the brittleness of the lineinfile and decided to fix the broader issue in https://openedx.atlassian.net/browse/OPS-3341

We will just remove the rc.local addition in bionic for now

**Review by coryleeio — APPROVED** [review]

**Pull request merged**